### ISSUE TITLE
Form Group - Group labels should use div instead of label

### ISSUE THREAD

**Issue by orion-cengage** [cengage/react-magma#1263]
CENG_3324

Docs: https://react-magma.cengage.com/version/4.3.0/api/formgroup/

Success Criteria: https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

The non-heading group labels are structured using the <label> element (e.g., <label class="css-en7bd3" id="52472c95-1027-445d-9654-b8170c91abe5">Choose one or more colors</label>), but the <label> is not associated with a matching form control, such as a checkbox.  Rather than using the <label> element, the best practice would be to replace it with a <div>.
